django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -75,15 +75,15 @@ Requires core.js, SelectBox.js and addevent.js.
             filter_input.id = field_id + '_input';
 
             selector_available.appendChild(from_box);
-            var choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', 'javascript:void(0);', 'id', field_id + '_add_all_link');
+            var choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', '#javascriptvoid', 'id', field_id + '_add_all_link');
```

Comment:
I couldn't find a reference for the "#javascriptvoid" technique (searching GitHub and Google). Could you give or link an explanation? Is the idea that it just needs to be a fragment that doesn't exist?